manifests/config.pp

```diff
@@ -8,8 +8,8 @@
 # @api private
 #
 class selinux::config (
-  $mode,
-  $type,
+  Optional[Enum['enforcing', 'permissive', 'disabled']] $mode = undef,
+  Optional[Enum['targeted', 'minimum', 'mls']] $type          = undef,
```

Comment:
Isn't this an API change? Previously it was a required parameter, now it becomes optional.
```suggestion
  Enum['enforcing', 'permissive', 'disabled'] $mode,
  Enum['targeted', 'minimum', 'mls'] $type,
```